met-web/src/components/layout/Drawer/CustomDrawer.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import AppBar from '@mui/material/AppBar';
+import { Grid, ListItemButton, List, ListItem, ListItemText, Box } from '@mui/material';
+import { useAppSelector } from '../../../hooks';
+import UserService from '../../../services/UserServices';
+
+
+const list = () => (
+    <Box
+        sx={{
+            background: '#003366',
+            height: '100%',
+            color: 'white',
+            width: 'auto',
+            justifyContent: 'center',
+            alignItems: 'center',
+        }}
+        role="presentation"
+    >
+        <List>
+            {['Engagements', 'Surveys', 'Calendar', 'Reporting'].map((text, index) => (
```

Comment:
the links here should be customizable and not hard coded. They could pass as props to the drawer component. We can define them in another file where we can change them if needed. Also They should pass as objects { name: "engagement", path: "/engagement",...}